packages/cli/src/options/beaconNodeOptions/chain.ts

```diff
@@ -285,6 +285,8 @@ Will double processing times. Use only for debugging purposes.",
   },
 
   "chain.archiveDataEpochs": {
+    // TODO: remove this alias after Lodestar 2.0
+    alias: "chain.archiveBlobEpochs",
```

Comment:
this todo seems unnecessary, we have a lot of `alias` and they are easy to find
```suggestion
    alias: "chain.archiveBlobEpochs",
```